diff --git a/third_party/blink/tools/blinkpy/web_tests/controllers/manager.py b/third_party/blink/tools/blinkpy/web_tests/controllers/manager.py
index 693a95d..7890234 100644
--- a/third_party/blink/tools/blinkpy/web_tests/controllers/manager.py
+++ b/third_party/blink/tools/blinkpy/web_tests/controllers/manager.py
@@ -452,9 +452,9 @@ class Manager(object):
         if (self._port.requires_http_server()
                 or any(self._is_http_test(test) for test in tests_to_run)):
             self._printer.write_update('Starting HTTP server ...')
-            self._port.start_http_server(
-                additional_dirs={},
-                number_of_drivers=self._options.max_locked_shards)
+            # self._port.start_http_server(
+            #     additional_dirs={},
+            #     number_of_drivers=self._options.max_locked_shards)
             self._http_server_started = True
 
         if any(self._is_websocket_test(test) for test in tests_to_run):
diff --git a/third_party/blink/tools/blinkpy/web_tests/port/base.py b/third_party/blink/tools/blinkpy/web_tests/port/base.py
index 6f5b188..0ae630e 100644
--- a/third_party/blink/tools/blinkpy/web_tests/port/base.py
+++ b/third_party/blink/tools/blinkpy/web_tests/port/base.py
@@ -495,15 +495,15 @@ class Port(object):
                 self.get_option('configuration')):
             return exit_codes.UNEXPECTED_ERROR_EXIT_STATUS
 
-        if not self._check_file_exists(self._path_to_image_diff(),
-                                       'image_diff'):
-            return exit_codes.UNEXPECTED_ERROR_EXIT_STATUS
+        # if not self._check_file_exists(self._path_to_image_diff(),
+        #                                'image_diff'):
+        #     return exit_codes.UNEXPECTED_ERROR_EXIT_STATUS
 
         if self._dump_reader and not self._dump_reader.check_is_functional():
             return exit_codes.UNEXPECTED_ERROR_EXIT_STATUS
 
-        if needs_http and not self.check_httpd():
-            return exit_codes.UNEXPECTED_ERROR_EXIT_STATUS
+        # if needs_http and not self.check_httpd():
+        #     return exit_codes.UNEXPECTED_ERROR_EXIT_STATUS
 
         return exit_codes.OK_EXIT_STATUS
 
