diff --git a/third_party/blink/tools/blinkpy/web_tests/port/base.py b/third_party/blink/tools/blinkpy/web_tests/port/base.py
index 6f5b188..0ae630e 100644
--- a/third_party/blink/tools/blinkpy/web_tests/port/base.py
+++ b/third_party/blink/tools/blinkpy/web_tests/port/base.py
@@ -495,15 +495,15 @@ class Port(object):
                 self.get_option('configuration')):
             return exit_codes.UNEXPECTED_ERROR_EXIT_STATUS
 
-        if not self._check_file_exists(self._path_to_image_diff(),
-                                       'image_diff'):
-            return exit_codes.UNEXPECTED_ERROR_EXIT_STATUS
+        # if not self._check_file_exists(self._path_to_image_diff(),
+        #                                'image_diff'):
+        #     return exit_codes.UNEXPECTED_ERROR_EXIT_STATUS
 
         if self._dump_reader and not self._dump_reader.check_is_functional():
             return exit_codes.UNEXPECTED_ERROR_EXIT_STATUS
 
-        if needs_http and not self.check_httpd():
-            return exit_codes.UNEXPECTED_ERROR_EXIT_STATUS
+        # if needs_http and not self.check_httpd():
+        #     return exit_codes.UNEXPECTED_ERROR_EXIT_STATUS
 
         return exit_codes.OK_EXIT_STATUS
 
diff --git a/third_party/blink/tools/blinkpy/web_tests/port/mac.py b/third_party/blink/tools/blinkpy/web_tests/port/mac.py
index 8b69240..6d608c3 100644
--- a/third_party/blink/tools/blinkpy/web_tests/port/mac.py
+++ b/third_party/blink/tools/blinkpy/web_tests/port/mac.py
@@ -93,8 +93,9 @@ class MacPort(base.Port):
     #
 
     def path_to_apache(self):
-        return self._path_from_chromium_base('third_party', 'apache-mac',
-                                             'bin', 'httpd')
+        # return self._path_from_chromium_base('third_party', 'apache-mac',
+        #                                      'bin', 'httpd')
+        return 'httpd'
 
     def path_to_apache_config_file(self):
         config_file_basename = 'apache2-httpd-%s-php7.conf' % (
diff --git a/third_party/blink/tools/blinkpy/web_tests/servers/apache_http.py b/third_party/blink/tools/blinkpy/web_tests/servers/apache_http.py
index 8a253db..e0c6cf5 100644
--- a/third_party/blink/tools/blinkpy/web_tests/servers/apache_http.py
+++ b/third_party/blink/tools/blinkpy/web_tests/servers/apache_http.py
@@ -91,14 +91,14 @@ class ApacheHTTP(server_base.ServerBase):
             "'%s' does not exist." % output_dir
         assert self._filesystem.exists(inspector_sources_dir), \
             "'%s' does not exist." % inspector_sources_dir
-        assert self._filesystem.exists(generated_sources_dir), \
-            "'%s' does not exist." % generated_sources_dir
+        # assert self._filesystem.exists(generated_sources_dir), \
+        #     "'%s' does not exist." % generated_sources_dir
 
         # yapf: disable
         start_cmd = [
             executable,
             '-f', '%s' % self._port_obj.path_to_apache_config_file(),
-            '-C', 'ServerRoot "%s"' % self._port_obj.apache_server_root(),
+            # '-C', 'ServerRoot "%s"' % self._port_obj.apache_server_root(),
             '-C', 'DocumentRoot "%s"' % document_root,
             '-c', 'Alias /js-test-resources "%s/resources"' % test_dir,
             '-c', 'Alias /geolocation-api/js-test-resources "%s/geolocation-api/resources"' % test_dir,
@@ -144,8 +144,8 @@ class ApacheHTTP(server_base.ServerBase):
                 '-k', 'start'
             ]
 
-        if self._port_obj.http_server_requires_http_protocol_options_unsafe():
-            start_cmd += ['-c', 'HttpProtocolOptions Unsafe']
+        # if self._port_obj.http_server_requires_http_protocol_options_unsafe():
+        #     start_cmd += ['-c', 'HttpProtocolOptions Unsafe']
 
         enable_ipv6 = self._port_obj.http_server_supports_ipv6()
         # Perform part of the checks Apache's APR does when trying to listen to
@@ -224,7 +224,7 @@ class ApacheHTTP(server_base.ServerBase):
         proc = self._executive.popen([
             self._port_obj.path_to_apache(), '-f',
             self._port_obj.path_to_apache_config_file(), '-C',
-            'ServerRoot "%s"' % self._port_obj.apache_server_root(), '-c',
+            # 'ServerRoot "%s"' % self._port_obj.apache_server_root(), '-c',
             'PidFile "%s"' % self._pid_file, '-k', 'stop'
         ])
         _, err = proc.communicate()
